packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt

```diff
@@ -60,6 +64,29 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
 
     init {
         viewWeakReference = WeakReference(view)
+
+        view.setFontAssetDelegate(object : FontAssetDelegate() {
+            override fun fetchFont(fontFamily: String): Typeface {
+                val typeface = ReactFontManager.getInstance()
+                    .getTypeface(fontFamily, UNSET, UNSET, view.context.assets)
+                return typeface?: Typeface.create("Roboto", 400)
```

Comment:
We need to find a way to dynamically set font delegate. Do you think you can improve the API instead?

Reply:
* Sorry I should use default typeface instead in this line, I'll update it.

Can you explain more about dynamically set font delegate?

Reply:
Now that you changed it to default typeface it makes more sense. No need for that. How about iOS side?

Reply:
React Native uses the CoreText API in iOS, so there is no need to set an [AnimationFontProvider](https://github.com/airbnb/lottie-ios/blob/cdf8499996eec658e62d1a116c1d87853d563101/Sources/Public/FontProvider/AnimationFontProvider.swift#L17).